src/nrniv/nrnbbcore_write.cpp

```diff
@@ -1971,10 +1971,45 @@ int nrncore_run(const char* arg) {
 #endif
   return r(nrn_nthread, have_gap, nrnmpi_use, nrn_use_fast_imem, arg);
 }
-#else
+
+int (*nrnpy_nrncore_enable_value_p_)();
+/** Return neuron.coreneuron.enable */
+int nrncore_is_enabled() {
+  if (nrnpy_nrncore_enable_value_p_) {
+    int b = (*nrnpy_nrncore_enable_value_p_)();
```

Comment:
just a nitpick : this is very simple case but just to mention : in our internal code review we are suggesting to not to use single char var like a, b but something slightly elaborate. for example state, enabled etc. :relaxed:

Reply:
Changed b to enable